liquidcore/config/serializers.py

```diff
@@ -117,3 +119,25 @@ class RegistrationSerializer(serializers.Serializer):
     lan = LanSerializer()
     wan = WanSerializer()
     ssh = SshSerializer()
+
+class VPNClientKeySerializer(serializers.ModelSerializer):
+    revoked_by = serializers.ReadOnlyField(source='revoked_by.username')
+    def to_representation(self, instance):
+        """Omit null fields from the model"""
+        # stolen from https://stackoverflow.com/a/45569581
```

Comment:
Do we need this comment? Maybe add it to the commit message instead?

Reply:
Good idea.